src/menu.cpp

```diff
@@ -13389,6 +13390,7 @@ void buttonStartSingleplayer(button_t* my)
 	numplayers = 0;
 	introstage = 3;
 	fadeout = true;
+	MainMenu::beginFade(MainMenu::FadeDestination::GameStart);

```

Comment:
Not sure how safe it is to interleave new menu code in the old menu code. I assume all of this old menu code will eventually be superseded by the new menu code, and situations like these are just temporary as we continue in-dev.

Reply:
Correct the old death prompt (purple panel/buttons) will need to be upgraded and then will use the correct new functions